jobtastic/tests/test_broker_fallbacks.py

```diff
@@ -148,20 +156,20 @@ def test_delay_or_eager_bad_channel(self):
 )
 
 
-class WorkingBrokerTestCase(AppCase):
-    def setup(self):
+class WorkingBrokerTestCase(TestCase):
```

Comment:
In newer celery, `self.app` is not being respected. For now commit to using the django `TestCase`. When we stop using django to run the tests we can figure out how to do this better.